File: app/src/main/java/com/example/ign_app/videomodel/videodata/VideoAssets.java
package com.example.ign_app.videomodel.videodata;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class VideoAssets {
    @SerializedName("url")
    @Expose
    private String url;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    @Override
    public String toString() {
        return "Assets{" +
                "url='" + url + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/ign_app/videomodel/videodata/VideoAssets.kt
package com.example.ign_app.videomodel.videodata

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class VideoAssets {
    @SerializedName("url")
    @Expose
    var url: String? = null

    override fun toString(): String {
        return "Assets{" +
                "url='" + url + '\'' +
                '}'
    }
}